src/bootstrap/bootstrap.py

```diff
@@ -664,7 +664,7 @@ def maybe_download_ci_toolchain(self):
         if self.verbose:
             print("using downloaded stage1 artifacts from CI (commit {})".format(commit))
         self.rustc_commit = commit
-        # FIXME: support downloading artifacts from the beta channel
+        # NOTE: CI artifacts are stored under nightly even when this is run from the beta branch
```

Comment:
This isn't true once beta has branched a while back I think; the last bors commit may be either beta or nightly, and it's hard to tell -- I guess we can look if that commit's src/ci/run.sh has a release-channel=beta but that feels pretty annoying.

Reply:
Hmm, I'm not sure I understand when commits show up under beta - is it when a beta backport PR lands?

I guess the way to fix it is to check /beta if /nightly 404s? That seems simple enough; not sure if I understand this well enough to extend it to stable thouhg.